sdk/storage/storage-blob/src/generated/src/models/index.ts

```diff
@@ -2100,6 +2099,11 @@ export interface PageBlobCreateOptionalParams extends coreHttp.RequestOptionsBas
    * Timeouts for Blob Service Operations.</a>
    */
   timeoutInSeconds?: number;
+  /**
+   * Optional. Indicates the tier to be set on the page blob. Possible values include: 'P4', 'P6',
+   * 'P10', 'P15', 'P20', 'P30', 'P40', 'P50', 'P60', 'P70', 'P80'
+   */
+  tier?: PremiumPageBlobAccessTier;
```

Comment:
This needs a transform back to AccessTier

Reply:
Done.